src/main/java/Duke.java

```diff
@@ -1,10 +1,236 @@
+     * @param lineDivider a string for line separator.
+     * @return error which is a boolean that tell us if there is an error or not.
+     */
+    public static boolean hasUnmarkTaskError(int unMarkIndex, String lineDivider){
+        boolean error = false;
+        try {
+            assignments[unMarkIndex].unmarkAsDone();
+        } catch (NullPointerException e) {
+            System.out.println("\t You are trying to unmark a task that has not been specified!");
+            System.out.println("\t" + lineDivider);
+            error = true;
+        }
+        return error;
+    }
+
+    /**
+     * Prints the starting message of the CLI.
+     *
+     * @return linDivider a string for line separator
+     */
+    public static String printWelcomeMessage() {
+        String logo = "___________.__                ___.\n"
+                + "\\_   _____/|  | _____    _____|  |__\n"
+                + " |    __)  |  | \\__  \\  /  ___/  |  \\\n"
+                + " |     \\   |  |__/ __ \\_\\___ \\|   Y  \\\n"
+                + " \\___  /   |____(____  /____  >___|  /\n"
+                + "     \\/              \\/     \\/     \\/\n";
+        System.out.println("Hello from\n"+ logo);
+        String lineDivider = "____________________________________________________________\n";
+        System.out.println("\t" + lineDivider
+                + "\t Hello! I'm Flash\n"
+                + "\t What can I do for you?\n"
+                + "\t" + lineDivider);
+        return lineDivider;
+    }
+
+    /**
+     * Prints out a list of the tasks saved from the user inputs.
+     */
+    public static void printList() {
+        int numberOrder = 1;
+
+        System.out.println("\t Here are the tasks in your list:");
+
+        for (int i = 0; i < indexTask; i++) {
+            System.out.println("\t " + numberOrder + ".["
+                    + assignments[i].getStatusOfTypeTask() + "]["
+                    + assignments[i].getStatusOfDone() + "] "
+                    + assignments[i].displayTypeTaskDetails());
+            numberOrder++;
+        }
+    }
+
+    /**
+     * Prints a message in CLI to inform the user if the task has been marked or not.
+     *
+     * @param markIndex index of which task to mark in splitUserInput[1].
+     * @param isMark boolean true or false to check if the task has been marked.
+     */
+    public static void printMarkOrUnmarkTask(int markIndex, boolean isMark) {
+        if (isMark) {
+            System.out.println("\t Awesome! I've marked this task as done:");
+        } else {
+            System.out.println("\t Awesome! I've marked this task as not done yet:");
+        }
+        System.out.println("\t\t [" + assignments[markIndex].getStatusOfTypeTask()
+                + "]" + "[" + assignments[markIndex].getStatusOfDone()
+                + "] " + assignments[markIndex].description);
+    }
+
     public static void main(String[] args) {
```

Comment:
Please consider avoiding long methods with more abstraction. 

As stated on the module website, 
> Be wary when a method is longer than the computer screen, and take corrective action when it goes beyond 30 LOC (lines of code). The bigger the haystack, the harder it is to find a needle.